### PR TITLE
Fix bug #122 & #124

### LINKED ISSUE #122: [BUG] - ArgumentError: comparison of Array with Array failed
**Describe the bug**
When processing the "sort alphabetically" method if the list has any record that contains a dot character, the program throws an exception.

**To Reproduce**
1. run `rails c`
3. run `TurkishCities.list_neighborhoods("İzmir", "Konak")`
4. program throws => 
```
ArgumentError: comparison of Array with Array failed
from /Users/username/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/turkish_cities-0.7.0/lib/turkish_cities/helpers/decomposer_helper.rb:80:in `sort_by'
```

**Expected behavior**
Users can see the neighborhoods of İzmir, Konak.
Example output:
```json
[
  "1.kadriye Mah",
  "19 Mayıs Mah",
  "2.kadriye Mah",
  "26 Ağustos Mah"
]
```

### LINKED ISSUE #124: [BUG] - ArgumentError: comparison of Array with Array failed
**Describe the bug**
When processing the "sort alphabetically" method if the list has any record that contains a dot character, the program throws an exception.

**To Reproduce**
1. run `rails c`
3. run `TurkishCities.list_neighborhoods('Artvin', 'Yusufeli')`
4. program throws => 
```
ArgumentError: comparison of Array with Array failed
from /Users/username/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/turkish_cities-0.7.0/lib/turkish_cities/helpers/decomposer_helper.rb:80:in `sort_by'
```

**Expected behavior**
Users can see the neighborhoods of İzmir, Konak.
Example output:
```json
[
...
"Abanoz Mah (Bahçeli Köyü)",
 "Abramet -kakrazevl Mah (Altıparmak Köyü)",
 "Ada Mah (Çevreli Köyü)",
 "Ada Mah (Irmakyanı Köyü)",
...
]
```

### PR DESCRIPTION
**Pull request by atagunay**
### Related github issue for this PR

Resolves #122 
Resolves #124 

### Checklist

* [x] I have performed a self-review of my own code
* [x] I added comments, particularly in hard-to-understand areas
* [x] I have made corresponding changes to the documentation `README.md`
* [x] I added tests that prove my fix is effective or that my feature works
* [x] `bundle exec rake` passes locally
* [x] `rubocop` gives no error locally
* [x] My PR title includes "WIP" or is [draft PR](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests#draft-pull-requests) if work is in progress

### Description
I added new character for sorting char array.

### Type of change

* Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: lib/turkish_cities/helpers/decomposer_helper.rb [hunk 1]
```diff
@@ -75,7 +75,7 @@ def prepare_city_list(city_list, options)
   end
 
   def sort_alphabetically(list, options = nil)
-    turkish_alphabet = ' -0123456789abcçdefgğhıijklmnoöprsştuüvyz'
+    turkish_alphabet = ' -.0123456789abcçdefgğhıijklmnoöprsştuüvyz'
```
Comment (sarslanoglu) [inline lib/turkish_cities/helpers/decomposer_helper.rb:78]: Nice catch 👍

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 ## 0.7.0 (2022-08-03)
 
+### Bug fixes
```
Comment (sarslanoglu) [inline CHANGELOG.md:5]: This is not going to be included to version 0.7.0 Can you do something like this?
```suggestion
# Changelog

## Unreleased version

### Bug fixes

* [#122](https://github.com/sarslanoglu/turkish_cities/issues/122): ArgumentError: comparison of Array with Array failed

## 0.7.0 (2022-08-03)

### New features
...
```

File: lib/turkish_cities/helpers/decomposer_helper.rb (hunk 1 above)
Comment (atagunay) [reply lib/turkish_cities/helpers/decomposer_helper.rb:78]: Thanks :)

**Review by sarslanoglu — APPROVED** [review]

**Review by sarslanoglu — APPROVED** [review]
🚀

**Pull request merged**